### PR TITLE
mysql: migrate slow queries

### PR DESCRIPTION
**Pull request by laysauchoa**
Migration: https://help.aiven.io/en/articles/4998219-logging-slow-queries-in-mysql

Migrating article from help aiven io to devportal

### CODE REVIEW

File: docs/products/mysql/howto/enable-slow-queries.rst [hunk 1]
```diff
@@ -0,0 +1,64 @@
+Enable slow queries
```
Comment (ftisiot) [inline docs/products/mysql/howto/enable-slow-queries.rst:1]:
```suggestion
Enable slow query logging
```

File: docs/products/mysql/howto/enable-slow-queries.rst [hunk 2]
```diff
@@ -0,0 +1,64 @@
+Enable slow queries 
+===================
+
+You can identify inefficient or time-consuming queries by enabling `slow queries <https://dev.mysql.com/doc/refman/5.7/en/slow-query-log.html>`_ in your MySQL service. In this tutorial, you can find how can you enable slow queries in your Aiven for MySQL service.
```
Comment (ftisiot) [inline docs/products/mysql/howto/enable-slow-queries.rst:4]:
```suggestion
You can identify inefficient or time-consuming queries by enabling `slow query log <https://dev.mysql.com/doc/refman/5.7/en/slow-query-log.html>`_ in your MySQL service. In the following sections you can find how can you enable slow queries in your Aiven for MySQL service.
```

File: docs/products/mysql/howto/enable-slow-queries.rst [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+Prerequisites
+-------------
+
+* An Aiven account with an Aiven for MySQL service running.
+
+Configure slow queries
```
Comment (ftisiot) [inline docs/products/mysql/howto/enable-slow-queries.rst:11]:
```suggestion
Configure slow queries in the Aiven Console
```

File: docs/products/mysql/howto/enable-slow-queries.rst [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+----------------------
+
+Follow these steps to enable your slow queries in your Aiven for MySQL service:
```
Comment (ftisiot) [inline docs/products/mysql/howto/enable-slow-queries.rst:14]:
```suggestion
Follow these steps to enable your slow queries in your Aiven for MySQL service via the `Aiven Console <https://console.aiven.io/>`_:
```

File: docs/products/mysql/howto/enable-slow-queries.rst [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+
+1. On the **Overview** page, scroll down to the **Advanced configuration** section and click **Add configuration option**.
+2. Click **Add configuration option** and choose the ``slow_query_log``. 
+3. Enable ``slow_query_log`` by toggling it to On. By default, ``slow_query_log`` is disabled.
+4. Click **Add configuration option** and choose the ``long_query_time``. 
+5. Set ``long_query_time`` according to your use case. By default, the value is 10 seconds.
```
Comment (ftisiot) [inline docs/products/mysql/howto/enable-slow-queries.rst:20]:
```suggestion
5. Set ``long_query_time`` according to your specific need. By default, the value is 10 seconds.
```

**Review by ftisiot — CHANGES_REQUESTED** [review]
Thanks @laysauchoa 
Small changes to make the article and titles more informative

**Comment by laysauchoa** [comment]
Hi @ftisiot, I accepted all the changes, thank you!!!

**Review by ftisiot — APPROVED** [review]
LGTM

**Pull request merged**